Source/ChocolateyGui/Models/LogMessage.cs

```diff
@@ -1,23 +1,17 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright company="Chocolatey" file="StreamingLogMessage.cs">
+// <copyright company="Chocolatey" file="LogMessage.cs">
 //   Copyright 2014 - Present Rob Reynolds, the maintainers of Chocolatey, and RealDimensions Software, LLC
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-using System.Runtime.Serialization;
-
 namespace ChocolateyGui.Models
 {
-    [DataContract]
-    public class StreamingLogMessage
+    public class LogMessage

```

Comment:
Is this class strictly relevant anymore? With no more streaming, log messages should be able to go straight to the logger.

Reply:
Probably not, but I didn't know what to do with the requests (and it was late)

Reply:
👇